tests/test_filtering.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+from datetime import datetime, timedelta
+from typing import List
+
+from personal_finances.transaction.definition import SimpleTransaction
+from personal_finances.transaction.filtering import transaction_datetime_filter
+
+TEST_TRANSACTIONS_QTD_TO_GENERATE = 300
+TEST_TRANSACTION_START_DATETIME = datetime(1994, 1, 2, 3, 4, 5)
+TEST_TRANSACTION_END_DATETIME = TEST_TRANSACTION_START_DATETIME + timedelta(
+    seconds=TEST_TRANSACTIONS_QTD_TO_GENERATE
+)
+
+
+def create_simple_transaction(index: int, datetime: datetime) -> SimpleTransaction:
+    return SimpleTransaction(
+        transactionId="transaction_" + str(index),
+        datetime=datetime,
+        amount=index,
+        referenceText="reference_transaction",
+        bankTransactionCode="dummy_transaction_code",
+    )
+
+
+def create_transactions_in_range(
```

Comment:
This is creating a bunch of transactions with the same `transaction_datetime`, could we exercise a bit more the range? Otherwise your tests won't be testing the edges of the filter interval.

Reply:
Here I didn't get if you are talking about just the function scope or for the entire tests cases. If it's just the unit test scope, it is creating each transaction with a different datetime, with a step of 1 second. 

 `transaction_datetime = datetime_offset + timedelta(seconds=i)`

Reply:
Uh I need some glasses I misread `i` by `1`. Fair enough, you are exercising a little bit of the ranges, but it doesn't seem to be aware of the whole range. The final is arbitrary for this function. What about exercising the whole range including the edges?